esp/logging/loggingmanager/loggingmanager.cpp

```diff
@@ -256,11 +267,17 @@ bool CLoggingManager::updateLog(IEspContext* espContext, IEspUpdateLogRequestWra
         }
         else
         {
+            Owned<IPropertyTree> scriptValues = req.getScriptValuesTree();
             for (unsigned int x = 0; x < loggingAgentThreads.size(); x++)
             {
                 IUpdateLogThread* loggingThread = loggingAgentThreads[x];
                 if (loggingThread->hasService(LGSTUpdateLOG))
                 {
+                    //leave the fact that a script can control the thread queue as an option undocumented, naming scheme could change,
+                    //  controlling the queue is a very low level mechanism and should be frowned upon
+                    //  once scripts can communicate with the agent that should be the mechanism to skip a particular type of logging
+                    if (checkSkipThreadQueue(scriptValues, loggingThread->queryControlName()))
+                        continue;
```

Comment:
Why not do the same before the line 264? We may also need a way to do the same when the Failover is enabled.

Reply:
@wangkx Updated based on your suggestion.  I'm not that familiar with that code so perhaps when Terrence is done he can give you the config and you can modify it to test that use case?

Reply:
I like that because I will have a better config for testing.

You update looks fine. I only reviewed the logentry and logthread changes. @asselitx and @timothyklemm may review more.